packages/react/src/components/Select/Select.tsx

```diff
@@ -379,7 +379,9 @@ const Select = (props: SelectProps) => {
                 </>
               )}
               <input
-                aria-activedescendant={`${id}-${activeOption}`}
+                {...(activeOption && {
+                  'aria-activedescendant': `${id}-${activeOption}`,
+                })}
```

Comment:
I assumed it would include the attribute as undefined, but apparently react will remove it. There is no issue doing it the current way, but I agree that using a ternary is more readable.